hypervisor/include/common/hypercall.h

```diff
@@ -322,9 +322,23 @@ int64_t hcall_setup_sbuf(struct vm *vm, uint64_t param);
 
 int64_t hcall_get_cpu_pm_state(struct vm *vm, uint64_t cmd, uint64_t param);
 
+/**
+ * @defgroup trusty_hypercall Trusty Hypercalls
+ *
+ * This is a special group that includes all hypercalls
+ * related to Trusty
+ *
+ * @{
+ */
 
 /**
- * @brief Switch VCPU state between Normal/Secure World.
+ * @brief Switch vCPU state between Normal/Secure World.
+ *
+ * * Hypervisor uses the Secure Monitor Code (SMC) instruction to do
```

Comment:
As @bzhu5 suggested, remove this SMC word, I think replace with "Hypervisor uses this hypercall to do ..." is OK.

Reply:
OK, I have updated the PR (and made additional, minor modifications to the doxygen comments). This is what the latest version looks like:

![image](https://user-images.githubusercontent.com/2192801/42283832-ffe6c472-7faa-11e8-8b77-42c6de1e4c3d.png)

What do you think?

Reply:
That's great. Thanks very much.